### DISCUSSION TITLE
Webhook Error 400

### DISCUSSION THREAD

**Discussion by jpdelon** [btcpayserver/btcpayserver#6033] [Technical Support]
We receiving an Webhook 400 Error and I can't seem to find out where it comes from.

If I send a test event to a webhook endpoint everything works fine.
Just the normal Webhooks from the store being undelivered.

We also see that all Invoices are gone.
"There are no invoices matching your criteria. "

**Comment by mp21-org** [comment]
I've seen "HTTP 400 Bad Request" before when I was talking https to a http port or vise versa.
Have you specified the protocol (https://) in your webhook URL? Is the ssl on your webserver correctly configured?

If I remember correctly (might need to double check in the code on this one), http is only allowed for .onion URLs = TOR

**Reply by jpdelon** [reply]
Hey,

thanks for your fast reply.
We specified https in our webhook correct.
The SSL is correctly configured, the service was running now for over 1 1/2 year.
I copied the Webhook 1:1 in the BTCPay Settings and now we have one with a working green mark and one with a red cross having errors. There is still ID's being generated.
I also can't delete the faulty Webhook cause it's giving me an Error code

504 Gateway Time-out
nginx

**Reply by mp21-org** [reply]
webhook delete attempts timing out sounds like a bug to me. Can you post a seperate [issue](https://github.com/btcpayserver/btcpayserver/issues/new/choose) for that?

The same webhook config once working and once giving a 400 error sounds very odd. Can you also check if the is something regarding this in the server logs (server settings - logs) of your BTCPay Server?

**Reply by jpdelon** [reply]
Thanks for your reply.

Okay do you mean by posting a separate issue I should post a "Issue: 🐛 Bug Report" ?

I checked the server logs so far for our application and can't seem to find any error.
I am not that familiar with finding the BTCPAY logs, does the BTCPAY have it's own logs I can have a look into?

Here's a picture of boths Webhooks. Both have same details and infos as stated above.
![59](https://github.com/btcpayserver/btcpayserver/assets/126243877/b0a4546c-bff5-4f99-b71f-5f19721a1583)

**Reply by mp21-org** [reply]
Yes, a bug report would make sense.

BTCPay Server (for the admin user) has under Server Settings (lower left corner) and then under Logs a section where you can access the server logs.

Regarding your webhooks screenshot: Could you click on Modify on the second webhook (the one with the red cross) and scroll down to "recent deliveries". You can check the status codes of your deliveries and view their send messages. Maybe that brings some insight.

**Reply by jpdelon** [reply]
I will create this issue as you mentioned above.
Regarding the server settings I don't seem to find these settings with a log section. 
I am logged in Role "Owner".

I scrolled down to recent deliveries.
The status of each delivery is on "status	"Pending""
![26](https://github.com/btcpayserver/btcpayserver/assets/126243877/00bf9189-9b1a-48dc-9800-95697044cfa8)
